Provider: Default tags support (#66)
infobloxopen/terraform-provider-bloxone: docs/index.md

provider "bloxone" {
  csp_url = "https://csp.infoblox.com"
  api_key = "<BloxOne DDI API Key>"
  default_tags = {
    managed_by = "terraform"
    location   = "us-west-1"

>   }

Reviewer: I think we should have two blocks there. One with the basic config for csp_url and api_key. And another block that explains that default tags can be provided in the provider configuration as a way to define common tags and have this example under there.

Author: Instead, shall I add an `optional` comment above the default_tags section?

- `api_key` (String) API key for accessing the BloxOne API. Can also be configured by using the `BLOXONE_API_KEY` environment variable. https://docs.infoblox.com/space/BloxOneCloud/35430405/Configuring+User+API+Keys
- `csp_url` (String) URL for BloxOne Cloud Services Portal. Can also be configured using the `BLOXONE_CSP_URL` environment variable.

> - `default_tags` (Map of String) Default global tags the client can set for all requests.

Reviewer: ```suggestion
- `default_tags` (Map of String) Tags to default for all resources```

Author: Will make the change in provider.go schema
